js/ui/calendar/ui.calendar.js

```diff
@@ -1156,13 +1251,17 @@ const Calendar = Editor.inherit({
         delete this._viewWidthValue;
     },
 
+    _clearInlineWidth: function() {
+        this.$element().css('width', '');
+    },
+
     _disposeViews: function() {
-        this._view.$element().remove();
-        this._beforeView && this._beforeView.$element().remove();
-        this._afterView && this._afterView.$element().remove();
-        delete this._view;
-        delete this._beforeView;
-        delete this._afterView;
+        const views = [this._view, this._additionalView, this._beforeView, this._afterView];
+
+        for(let i = 0; i < views.length; i += 1) {
+            views[i]?.$element().remove();
+            delete views[i];
+        }
```

Comment:
we should also clear `this._skipNavigate` in `_dispose`

Reply:
done